From 86d0283d8af499ebe0bca1d82eb73b94f7418736 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 3 Aug 2020 11:37:42 +0800
Subject: [PATCH] Test case2 for Change build option under driver module

Description: Change build option under BuildOptions.common.EDKII.DXE_SMM_DRIVER, BuildOptions.common.EDKII.SMM_CORE (Add "-D MAX_RANK_DIMM=2") in TestPkg/buildoptions.dsc.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 TestPkg/buildoptions.dsc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/TestPkg/buildoptions.dsc b/TestPkg/buildoptions.dsc
index 9ca9e101ff..00ba7751c5 100644
--- a/TestPkg/buildoptions.dsc
+++ b/TestPkg/buildoptions.dsc
@@ -27,7 +27,7 @@
 # Force PE/COFF sections to be aligned at 4KB boundaries to support page level
 # protection of DXE_SMM_DRIVER/SMM_CORE modules
 [BuildOptions.common.EDKII.DXE_SMM_DRIVER, BuildOptions.common.EDKII.SMM_CORE]
-  GCC:*_*_*_DLINK_FLAGS = -z common-page-size=0x1000
+  GCC:*_*_*_DLINK_FLAGS = -z common-page-size=0x1000 -D MAX_RANK_DIMM=2
   XCODE:*_*_*_DLINK_FLAGS = -seg1addr 0x1000 -segalign 0x1000
   XCODE:*_*_*_MTOC_FLAGS = -align 0x1000
   CLANGPDB:*_*_*_DLINK_FLAGS = /ALIGN:4096
-- 
2.27.0.windows.1

